homeassistant/components/backup/manager.py

```diff
@@ -308,7 +335,12 @@ async def async_restore_backup(self, slug: str, **kwargs: Any) -> None:
         def _write_restore_file() -> None:
             """Write the restore file."""
             Path(self.hass.config.path(RESTORE_BACKUP_FILE)).write_text(
-                f"{backup.path.as_posix()};",
+                ";".join(
```

Comment:
This should change to dump a JSON dictionary, but that should be in a separate PR.